ush/soca/prep_ocean_obs_utils.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+import os
+import fnmatch
+import subprocess
+from wxflow import FileHandler, Logger
+
+logger = Logger()
+
+# finds files in DMPDIR matching the regex in an obtype's preobs config,
+# copies them to DATA, and returns a list of the files so handled
+
+
+def obs_fetch(config, runtime_config, obsprep_space, cycles):
+
+    DMPDIR = config.DMPDIR
+    COMIN_OBS = config.COMIN_OBS
+
+    RUN = runtime_config.RUN
+    PDY = runtime_config.PDY
+    cyc = runtime_config.cyc
+
+    subdir = obsprep_space['dmpdir subdir']
+    dumpdir_regex = obsprep_space['dmpdir regex']
+    matching_files = []
+    file_copy = []
+
+    for cycle in cycles:
+
+        PDY = cycle.strftime('%Y%m%d')
+        cyc = cycle.strftime('%H')
+
+        full_input_dir = os.path.join(DMPDIR, RUN + '.' + PDY, cyc, subdir)
```

Comment:
I think things like this would be cleaner if it was `f"{RUN}.{PDY}"`

Reply:
done